runebender-lib/src/widgets/sidebar.rs

```diff
@@ -63,13 +63,13 @@ fn selected_glyph_widget() -> impl Widget<GlyphDetail> {
         .with_child(
             Flex::row()
                 .with_child(
-                    Label::new("kern group")
+                    Label::dynamic(|d: &GlyphDetail, _| (*d.kern2_group).to_string())
```

Comment:
so getting the editing working should be a two step process:

1. use `EditableLabel`:
    ```rust
    EditableLabel::new(
        |s: &String, _: &_| s.to_string(),
        |s| Some(s.to_string())
    )
    .lens(GlyphDetail::kern1_group)
    ```

2. Add some controller that sends a message to rename the glyph when the data changes.

That _should_ get this working, I think?

Reply:
That's what I thought, though it involves writing two more dummy lenses in GlyphDetail I think.

Reply:
Yep, that sounds right. I don't love it, and also, it works,